localisation/00_zuwc_united_localisation.csv

```diff
@@ -2882,19 +2882,19 @@ summon_gorak_tul_desc;With our hold over Drustvar secure, we are now able to sum
 summon_gorak_tul_tooltip;�Y[From.GetBestName]�! summons the warband of Gorak Tul.;;�Y [From.GetBestName]�! beschw�rt die Krieger von Gorak Tul.;;;;;;;;;;;x
 heartsbane_coven_formed;The Heartsbane Coven has been formed;;Der Herzbannzirkel wurde gegr�ndet;;;;;;;;;;;x
 drust_opinion;[drust.GetName] Opinion;;[drust.GetName] Meinung;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+laid_eggs;Laid Eggs;;;;;;;;;;;;;x
+laid_eggs_desc;This character gets these bonuses and maluses per born child due to their government type.;;;;;;;;;;;;;x
```

Comment:
I'm not too fond of the 'per born child' part of the sentence. My suggestion below could work, or you could replace it with 'per every child'. 
```suggestion
laid_eggs_desc;This character receives these bonuses and maluses per child due to their government type.;;;;;;;;;;;;;x
```